runtime/src/lib.rs

```diff
@@ -130,7 +130,8 @@ impl SignedExtension for OnlyStakingAndClaims {
 	{
 		match call {
 			Call::Staking(_) | Call::Claims(_) | Call::Sudo(_) | Call::Session(_)
-				| Call::ElectionsPhragmen(_)
+				| Call::ElectionsPhragmen(_) | Call::TechnicalMembership(_)
+				| Call::TechnicalCommittee(_)
```

Comment:
Do we need to enable `Call::Council(_)` as well?

Reply:
Nevermind, I see that getting into the council will go through `ElectionsPhragmen`

Reply:
not yet - we can do that once we actually want to have the council do stuff. but i'd prefer to test things one at a time first.